content/installations/install-visual-studio/_index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Visual Studio Code Installation"
+date: 2021-10-01T09:28:27-05:00
+draft: false
+weight: 10
+---
+
+## Setting Up Visual Studio Code
+
+Before we start coding on our computer, we need to make sure we have the right tools! Programmers use **integrated development environments** (IDE) to write and run their code. The development environment we will be using for this class is Visual Studio Code
+.
+In addition to simply writing and running code, Visual Studio Code has tools that recognize errors in our code and has an integrated terminal so we can navigate through our filesystem to find the files that need our attention.
+
+{{% notice blue Note "rocket" %}}
+Visual Studio Code is very customizable. Once you have everything set up, you can take additional steps to personalize your workspace such as changing the theme.
+{{% /notice %}}
+
+Go to the Visual Studio Code [download page](https://code.visualstudio.com/download/) and download the appropriate version for your operating system.
+
```

Comment:
Suggestion add a note for Mac use to make sure to download the correct version depending on the chip your mac has.  M1 and M2 are silicon chips.

Reply:
Added note within book for this.